api/src/main/java/org/openmrs/module/patientflags/task/PatientFlagTask.java

```diff
@@ -126,8 +126,22 @@ public void run() {
 			FlagService flagService = Context.getService(FlagService.class);
 			
 			for (Flag flag : flagService.getAllFlags()) {
-				generatePatientFlags(flag, flagService);
+				Daemon.runInNewDaemonThread(new PatientFlagGenerator(flag));
```

Comment:
Can we use streams instead of a for-loop?

Reply:
I updated the code